### ISSUE TITLE
Monitor bitbucket project/repo - auto create build plans

### ISSUE THREAD

**Issue by sbhakta99** [mibexsoftware/bamboo-plan-dsl-plugin#77]
Can this plugin monitor a Bitbucket Project, so whenever a new repo is checked in, it will auto create all build plans and deployment plans relative to that project?

**Comment by mrueegg** [comment]
No, unfortunately not. AFAIK there is no way to detect that a reposiory is created in Bitbucket Server from Bamboo. I think the only solution would be to have a Bitbucket plug-in that recognizes this event and triggers some action in Bamboo (e.g., creating deployment projects, plans etc. with the Bamboo REST API). Would that be an option?

Thanks for your understanding.

Best regards,
Michael

**Comment by sbhakta99** [comment]
That is an option, but there a quick way to import bitbucket repos/slugs using bitbucket apis?

We want to have Bamboo DSL job scan a Bitbucket Project for new repos, once a new repo is checked it we want DSL job to run to create that repos Bamboo Plans.
